apps/stripe/src/modules/stripe/stripe-publishable-key.ts

```diff
@@ -28,6 +28,18 @@ export class StripePublishableKey {
     );
   }
 
+  static createFromPersistedData(args: { publishableKeyValue: string }) {
+    if (StripePublishableKey.isInProperFormat(args.publishableKeyValue)) {
+      return ok(new StripePublishableKey(args.publishableKeyValue));
+    }
```

Comment:
this is leaky. I shouldn't have to remember to run validation before I create instance.

I should be able to do this

```
const result = StripePublishableKey.createFromString(...).match(// handle error)

```